sdk/lib/opentelemetry/sdk/resources/resource.rb

```diff
@@ -15,19 +15,23 @@ class << self
 
           # Returns a newly created {Resource} with the specified attributes
           #
-          # @param [Hash{String => String, Numeric, Boolean} attributes Hash of key-value pairs to be used
+          # @param [Hash{String => String, Numeric, Boolean}] attributes Hash of key-value pairs to be used
           #   as attributes for this resource
+          # @param [String] Specifies the Schema URL that should be recorded in the emitted resource.
           # @raise [ArgumentError] If attribute keys and values are not strings
+          # @raise [ArgumentError] If the schema URL is given but it is not a string.
           # @return [Resource]
-          def create(attributes = {})
+          def create(attributes = {}, schema_url = nil)
```

Comment:
Can we make this a named parameter as well?
```suggestion
          def create(attributes = {}, schema_url: nil)
```

Reply:
It [made the tests unhappy](https://github.com/open-telemetry/opentelemetry-ruby/pull/1237#discussion_r879757292), but I do _want_ to. I'll try a little harder.

Reply:
@fbogsany I implemented that here: 0966e37b - take a look and let me know if you hate it or not. I don't know quite how I feel.

Reply:
```
Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.
```

It's hard to overstate just how annoying the cop can be. I'll fix that later.